pkg/cmd/util_content_test.go

```diff
@@ -18,15 +18,20 @@ limitations under the License.
 package cmd
 
 import (
+	"context"
+	"fmt"
 	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
 	"os"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
 func TestRawContentFileMissing(t *testing.T) {
-	_, _, err := loadRawContent("dsadas")
+	_, _, err := loadRawContent(context.TODO(), "dsadas")
```

Comment:
I think `context.Background()` can be used for test purposes.

Reply:
done